torch/utils/_config_module.py

```diff
@@ -57,10 +57,14 @@ class _Config(Generic[T]):
         alias: If set, the directly use the value of the alias.
         env_name_force: If set, this environment variable has precedence over
             everything after this.
+            If multiple env variables are given, the precendence order is from
```

Comment:
nit - might indicate that the first value set to either 0 or 1 wins (invalid values are ignored).